reporter/src/main/kotlin/reporters/NoticeTemplateReporter.kt

```diff
@@ -166,7 +167,20 @@ class NoticeTemplateReporter : Reporter {
     /**
      * A collection of helper functions for the Freemarker templates.
      */
-    class NoticeHelper(private val licenseConfiguration: LicenseConfiguration) {
+    class NoticeHelper(private val ortResult: OrtResult, private val licenseConfiguration: LicenseConfiguration) {
+        /**
+         * Filter [packages] that are a dependency of at least one of the provided [projects][projectIds].
+         */
+        @Suppress("UNUSED") // This function is used in the templates.
+        fun filterByProjects(
+            packages: List<PackageNoticeModel>,
```

Comment:
Please use `Collection` for the function parameters.

Reply:
Done.